tools/test_cs_equal_db.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python3
+
+
+from util.cassandra_stress import *
+from util.scylla_docker import *
+from cassandra.cluster import Cluster
+
+import argparse
+
+
+### This test populates two tables with cassandra-stress and cql-stress-cassandra-stress.
+### Then it checks whether these two tables are equal.
+def run(args, node: ScyllaDockerNode, cs: CassandraStress):
+    cluster = Cluster([node.ip], port=node.port)
+    session = cluster.connect()
+
+    # Populate two tables in two keyspaces.
+    ks_cassandra = "ks_cassandra"
+    ks_cqlstress = "ks_cqlstress"
+
+    print("\n=== Populating DB... ===\n")
+
+    print(f"\n=== Inserting {args.workload_size} rows to {ks_cassandra}.standard1 with cassandra-stress ===\n")
+    cs.run(command="write",
+           workload_size=args.workload_size,
+           node_ip=node.ip, keyspace=ks_cassandra,
+           threads=args.concurrency)
+
+    print(f"\n=== Inserting {args.workload_size} rows to {ks_cqlstress}.standard1 with cql-stress ===\n")
+    run_cql_stress(command="write",
+                   workload_size=args.workload_size,
+                   node_ip=node.ip, keyspace=ks_cqlstress,
+                   threads=args.concurrency)
+
+    # Check whether two tables are equal.
+    cs_rows = session.execute(f'select * from {ks_cassandra}.standard1')
+    cqlstress_rows = session.execute(f'select * from {ks_cqlstress}.standard1')
+
+    print("\n=== Comparing tables... ===\n")
+    try:
+        if cs_rows != cqlstress_rows:
+            print("Compared tables are not equal!!!")
+            print("\n=== cassandra-stress rows: ===\n")
+            for row in cs_rows:
+                print(row)
+            print("\n=== cql_stress rows: ===\n")
+            for row in cqlstress_rows:
+                print(row)
```

Comment:
I wonder whether we should always print this. In CI, we are only inserting a small number of rows, but developers might use a larger number of rows when running bigger tests locally.

Reply:
True that. I'll remove it, since we won't drop the keyspaces anyway.